content/pandas/concepts/dataframe/terms/resetindex/.~lock.Austin_Animal_Center_Intakes.csv#

```diff
@@ -0,0 +1 @@
+,scientifik,lincomp1,28.02.2023 11:52,file:///home/scientifik/.config/libreoffice/4;
```

Comment:
I believe I had that pdf opened in libreoffice as a spreadsheet when I committed my repository the first time which is why I think .lock.Austin_Animal_Center_Intakes.csv showed up in the first place. I removed the pdf from my resetindex folder so I think that should remove both of those files. If that doesn't remove them please let me know what I need to do to get rid of it because it doesn't show up as a file on my end anymore. I also made all the changes you suggested then pushed to the repository again.